### PR TITLE
Update .htaccess (php8+ and mod_lsapi)

### LINKED ISSUE #26569: PHP8 in the .htaccess
Should be moddified L51 or add a new block to consider `<IfModule mod_php8.c> ` 

https://github.com/nextcloud/server/blob/8ef920fdf90bc54d6f17134ebd80a71db2f9d8ea/.htaccess#L51-L58

### PR DESCRIPTION
**Pull request by solracsf**
- Add `mod_lsapi` (Cloudlinux) authorization headers
- Add `mod_php8` php_values
- Reformating for better lisibilty

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -56,6 +61,29 @@
     SetEnv htaccessWorking true
   </IfModule>
 </IfModule>
+
+<IfModule mod_php8.c>
```
Comment (marcos-guerrero) [inline .htaccess:65]: The name of the php module in php8 is mod_php.c

Comment (acsfer) [reply .htaccess]: Of course it is 🚶

**Comment by andyxheli** [comment]
Still waiting on this correct ?

**Review by Pytal — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]
LGTM

**Comment by Pytal** [comment]
@acsfer CI needs signoff
```bash
git rebase HEAD~2 --signoff
git push --force-with-lease origin update-htaccess
```

File: .htaccess (hunk 1 above)
Comment (MichaIng) [reply .htaccess]: Not always, or not only. The default PHP-FPM invocation config in my case on Debian contains:
```
<IfModule !mod_php8.c>
```
And the library file name is `libphp8.0.so`. So if `mod_php.c` works, then it is an alias somewhere. I'll do a test to assure it works here as well.

**Comment by Pytal** [comment]
@acsfer lint error on CI, needs a
```bash
composer run-script cs:fix
```

**Comment by nickvergessen** [comment]
rebased

**Comment by andyxheli** [comment]
This is still missing on NC 22.1.1

**Comment by andyxheli** [comment]
> /backport to stable22

Thank you!

**Comment by MichaIng** [comment]
If I'm not mistaken, also NC21 has PHP8.0 support, doesn't it?

**Comment by andyxheli** [comment]
> If I'm not mistaken, also NC21 has PHP8.0 support, doesn't it?
> 
> EDIT: Yes it is: https://github.com/nextcloud/server/blob/stable21/lib/versioncheck.php
> NC20 not yet: https://github.com/nextcloud/server/blob/stable20/lib/versioncheck.php

Yes it dose.

**Comment by MichaIng** [comment]
I mean also without PHP8.0 support, the `mod_lsapi` support may still be worth it and the `mod_php` condition does not hurt in any way and could also manually be removed from the commit. But since half of the purpose is gone, I'll leave it up to others to create that backport.

**Pull request merged**